OpenRA.Mods.Common/Traits/Husk.cs

```diff
@@ -99,11 +115,35 @@ void INotifyCreated.Created(Actor self)
 		{
 			var distance = (finalPosition - CenterPosition).Length;
 			if (dragSpeed > 0 && distance > 0)
+			{
+				CrushAction(self, (notifyCrushed) => notifyCrushed.WarnCrush);
 				self.QueueActivity(new Drag(self, CenterPosition, finalPosition, distance / dragSpeed));
+			}
+
+			self.QueueActivity(new CallFunc(() =>
```

Comment:
There's a long term plan to get rid of `CallFunc` since its not compatible with what we want to do with real save games. Can we create a proper `HuskMove` activity that replaces `Drag` and `CallFunc` instead?

Reply:
I see

Reply:
Wouldn't it be better to modify drag to crush?

Reply:
It didn't make sense to make drag crush as it isn't tied to cells. Instead I created a `DragAndCrush` activity